CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ Remember to align the itemized text with the first line of an item within a list
     runtime for any reason, please let us know on the [JAX issue
     tracker](https://github.com/google/jax/issues).
 
+  * remove deprecated function `jax.numpy.msort`, which has been deprecated since
```

Comment:
Nit: this is under the wrong version (0.4.9 is next). Another PR should fix shortly.